.github/workflows/publish-documentation-gh-pages.yml

```
@@ -1,6 +1,8 @@
 name: Publish Documentation to GitHub Pages
 
 on:
+  merge_group:
+    types: [checks_requested]  # when a pull request is added to a merge queue
```

I think that is not needed.
```suggestion
    branches: [main]  # when a pull request is added to a merge queue
```

---

I think the branches argument might be needed since we only want to publish when merge targets main. Alternatively we could just use push?

---

yeah, seems like push is the simpler and more sensible solution for us.